types/handsontable/test/global.ts

```diff
@@ -0,0 +1,301 @@
+namespace HandsontableGlobalTest {
+  function test_HandsontableInit() {
+    const elem = document.createElement('div');
+    const hot = new Handsontable(elem, {
```

Comment:
the second declaration of `Handsontable` needs to be `declare var Handsontable: { ...`, not `interface Handsontable`. That's because `new` expects an actual object to be newed, not just a type. (This means that your class idea could be the right one, although I need to think about it a bit more.)